### PR TITLE
Homogenize codebase

### LINKED ISSUE #22: Homogenize codebases of offline-shell and offline-content
- [x] Establish a common naming pattern.
- [x] Common code guidelines. Do it by fixing #40 
- [x] Unify DB and option classes.
- [x] Merge redundant classes.

### PR DESCRIPTION
**Pull request by delapuente**
Fixes #22

### CODE REVIEW

**Comment by delapuente** [comment]
@darkwing, a quick review would be appreciated! I'm fixing tests right now. ;)

File: tests/test-shell.php [hunk 1]
```diff
@@ -69,13 +69,4 @@ function test_changed_file_list_generates_new_sw() {
 		$files[] = $last_file;
 		update_option('offline_shell_files', $files);
 	}
-
-	function test_update_runs_on_version_mismatch() {
```
Comment (delapuente) [inline tests/test-shell.php]: Per how the options class work, this is no longer needed.

**Comment by darkwing** [comment]
When I activated a different theme I got the following error:

```
PHP Fatal error:  Class 'Offline_Shell_DB' not found in /Applications/mampstack-5.5.28-0/apache2/htdocs/wp-content/plugins/wp-offline-content/class-wp-offline-shell-admin.php on line 82, referer: http://localhost:8080/wp-admin/themes.php
```

File: wp-offline-content/wp-offline-content.php [hunk 2]
```diff
@@ -1,40 +0,0 @@
-<?php
-/*
-Plugin Name: Offline Content
-Description: Allow your users to read your content even while offline.
-Plugin URI: https://github.com/delapuente/wp-offline-content
-Version: 0.6.1
```
Comment (darkwing) [inline wp-offline-content/wp-offline-content.php]: Says `0.6.1` here but `0.6.0` in other spots.

**Comment by darkwing** [comment]
Everything else appears to work OK!

**Pull request merged**